massa-execution/src/tests/scenarios_mandatories.rs

```diff
@@ -1,18 +1,110 @@
 // Copyright (c) 2021 MASSA LABS <info@massa.net>
 
-use crate::{start_controller, ExecutionConfig};
+use std::str::FromStr;
+
+use crate::{start_controller, ExecutionSettings, SCELedger, SCELedgerEntry};
+use massa_models::{address::AddressHashMap, Address, Amount, Slot};
+use massa_signature::{derive_public_key, generate_random_private_key};
 use serial_test::serial;
+use tempfile::NamedTempFile;
+
+/// generate a named temporary initial ledger file
+pub fn generate_ledger_bootstrap_file(values: &AddressHashMap<Amount>) -> NamedTempFile {
```

Comment:
Shouldn't that be renamed to generate_ledger_initial_file

Reply:
done